### DISCUSSION TITLE
https://plugin-md-enhance.vuejs.press/zh/guide/stylize/align.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#832] [Announcements]
# https://plugin-md-enhance.vuejs.press/zh/guide/stylize/align.html

自定义内容对齐方式。 配置 语法 演示

https://plugin-md-enhance.vuejs.press/zh/guide/stylize/align.html

**Comment by fengyacao** [comment]
居中好像对表格无效。
